tests/gene-comparison-tool.spec.ts

```diff
@@ -27,4 +27,46 @@ test.describe('specific viewport block', () => {
     const dropdown = page.locator('#subCategory');
     await expect(dropdown).toHaveText('Targeted Selected Reaction Monitoring (SRM)');
   });
+
+  test('switching from RNA to Protein with RNA-specific column ordering reverts back to Risk Score descending', async ({ page }) => {
+    // set category for Protein - Differential Expression
+    await page.goto('/genes/comparison?sortField=FP&sortOrder=1');
+
+    // wait for page to load (i.e. spinner to disappear)
+    await expect(page.locator('div:nth-child(4) > div > .spinner'))
+      .not.toBeVisible({ timeout: 150000});
+
+    // Gene Comparison Overview tutorial modal
+    const tutorialModal = page.getByText('Gene Comparison Overview');
+    await expect(tutorialModal).toBeVisible({ timeout: 10000});
+
+    // close the Gene Comparison Overview tutorial modal
+    const closeButton = page.locator('.p-dialog-header-close-icon');
+    await closeButton.click();
+
+    // Gene Comparison Overview tutorial modal
+    await expect(tutorialModal).not.toBeVisible({ timeout: 10000});
```

Comment:
Consider using another newly added helper method here as well?

```suggestion
    await closeGctHelpDialog(page);
```

Reply:
using new helper method